frontend/templates/frontend/flowchart.html

```diff
@@ -0,0 +1,902 @@
+			success: function (response) {
+				console.log("Success in submitting the input form")
+			},
+			error: function (response) {
+				console.log("Error in submitting the input form")
+			}
+		});
+	})
+
+
+	function submitFlowchart()
+	{
+		data = flowchart_id
+		if(flowchart_id!=undefined)
+		{
+			$.ajax({
+				data: data,
+				headers: {
+					"X-CSRFToken": '{{ csrf_token }}',
+				},
+	
+				type: "POST",
+				url: "/submit_flowchart/",
+				success: function (response) {
+					console.log("Success Flowchart")
+				},
+				error: function (response) {
+					console.log("Failure Flowchart")
+				}
+			});
+		}
+		else
+			alert("No flowchart to submit!")
+	}
+	
+
+	var input_calc_charge = null, input_calc_multiplicity = null, input_calc_resource = null, override_charge_multiplicity = null, override_resource = null
+	function submit_input() {
+		data = $("#inputform").serializeArray()
+		var override_charge_multiplicity_found = false;
+		var override_resource_found = false;
+		for(var i=0;i<data.length;i++)
+		{
+			if(data[i].name=="calc_charge")
+				input_calc_charge = data[i].value
+			if(data[i].name=="calc_multiplicity")
+				input_calc_multiplicity = data[i].value
+			if(data[i].name=="calc_resource")
+				input_calc_resource = data[i].value;
+			if(data[i].name=="override_flowchart_charge_multiplicity")
+			{
+				override_charge_multiplicity = data[i].value
+				override_charge_multiplicity_found = true;
+			}
+			else if(!override_charge_multiplicity_found)
+				override_charge_multiplicity = null;
+			if(data[i].name=="override_resource")
+			{
+				override_resource = data[i].value
+				override_resource_found = true;
+			}
+			else if(!override_resource_found)
+				override_resource=null;
+		}
+		{% if not ensemble and not structure and not calc%}
+		if (sketcher.molecules.length > 0) {
+			let mol = ChemDoodle.writeMOL(sketcher.getMolecule());
+			data.push({"name": "structure", "value": mol});
+		}
+		{% endif %}
+
+		upload_el = $("#file_structure");
+		if(upload_el.length != 0) {
+			num_files = upload_el[0].files.length;
+		}
+		data.push({"name": "num_files", "value": num_files});
+		$.ajax({
+			data: data,
+			headers: {
+				"X-CSRFToken": '{{ csrf_token }}',
+			},
+
+			type: "POST",
+			url: "/submit_flowchart_input/",
+			success: function (response) {
+				console.log("Success in submitting the flowchart form")
+				flowchart_order_id = response;
+			},
+			error: function (response) {
+				console.log("Error in submitting the flowchart form")
+			}
+		});
+		$(".modal-input").removeClass("is-active");
+	}
+
+	var para_calc_id
+	function verify_form_flowchart() {
+		data = $("#calcform").serializeArray()
+		form = $("#calcform");
+		formData = new FormData(form[0])
+		calc_para_map.set(parseInt(para_calc_id), formData)
+		data.push({"name": "constraint_num", "value": constraint_num});
+
+		{% if ensemble or structures %}
+		data.push({"name": "starting_ensemble", "value": {{ ensemble.id }}});
```

Comment:
The ids are not hashes, they need to be treated as strings within quotation marks

Reply:
Done!